src/docs/concepts-transactions/default-negotiation-process/index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Default negotiation process
```

Comment:
```suggestion
title: Negotiation process
```
I would drop "default" from the title. It's clearer and better for search indexing